executor/runtime/jobmaster/build.go

```diff
@@ -18,6 +30,44 @@ func RegisterBuilder() {
 
 type jobMasterBuilder struct{}
 
+func getEtcdMetaKV(ctx context.Context, clients *client.Manager) (metadata.MetaKV, error) {
```

Comment:
why we need expose etcd here,  metaserver interface is not enough?

Reply:
I have not figured out a good approach to get meta store from outside world, so here I get etcd address from master.